Sources/ErgastAPI/Models/Circuits/AllCircuits.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  AllCircuits.swift
+//
+//
+//  Created by Giovanni Noa on 5/5/20.
+//
+
+import Foundation
+
+public struct AllCircuits: Codable {
+    let data: AllCircuitData
+
+    enum CodingKeys: String, CodingKey {
+        case data = "MRData"
+    }
+}
+
+struct AllCircuitData: Codable {
+    let xmlns: String
+    let series: String
+    let url: String
+    let limit, offset, total: String
```

Comment:
Why are these vars grouped together but not the ones above? I think being consistent and declaring each on a separate line may be preferable.